test/helpers/builder.js

```diff
@@ -173,6 +170,15 @@ export class Builder {
   }
 }
 
+/** @param {import('multiformats').Link} cid */
+export const toCarKey = cid => `${cid}/${cid}.car`
+
+/** @param {import('multiformats').MultihashDigest} digest */
+export const toBlobKey = digest => {
+  const digestString = base58btc.encode(digest.bytes)
+  return `${digestString}/${digestString}.blob`
```

Comment:
Nit: I've been thinking whether doing something like `blob/${digestSstring}.blob` would be a better option ? At least visually it would be easier to distinguish that looking at car vs blob hashes and potentially other things in the future.

Reply:
@vasco-santos WDYT? We originally used `cid/cid.car` so that we could put indexes next to the data, like `cid/cid.car.idx` but we never did that and used a different bucket instead. So I'm not too bothered about having the extra "directory" per upload.

This would also be a good way to segregate new blob data from the old.

Reply:
We also do this keying with pathing to better optimize S3 Rate limits. Re R2 Rate limits, this was not rate limited like S3, but last time we talked (6 months + ago), they would be looking into having that possibility on their end. 

So, I would push back on doing this change, and prioritize to finish what we need on the reads interface to swap the bucket

Reply:
Also, we have different tables in Dynamo for Blobs and Store allocations. Therefore, if we ever need to go through the whole list, we can rely on dynamo instead of a list with prefix from bucket